packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTable.tsx

```diff
@@ -149,7 +224,8 @@ export function TransactionsTable({
   }, [locallyStoredTransactions])
 
   const status = (() => {
-    if (loading) return TableStatus.LOADING
+    if (loading || typeof isSmartContractWallet === 'undefined')
```

Comment:
With `isSmartContractWallet` defaulted to false, the UI would flash between states. Better to keep it loading when `isSmartContractWallet` is undefined.